ChallengeTestRunner/challenge_test_lib/challenge_test.py

```diff
@@ -265,11 +265,15 @@ def challenge_fun(self, *args, **kwargs):
     def assert_printed(self, value, index=None):
         str_value = str(value)
         if index is not None:
-            return str_value in self.mock_print.printed[index]
+            self.assertTrue(str_value in self.mock_print.printed[index],
```

Comment:
Please change to:
```python
if index is not None and str_value not in self.mock_print.printed[index]:
    raise AssertionError(msg)
```

So `assertTrue` does not show up in the stack trace, making it clearer.

Reply:
Done